Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.h

```diff
@@ -83,6 +84,15 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsCurveNode : public vt
   /// \return sum of distances between the curve points, returns 0 in case of an error
   double GetCurveLengthBetweenStartEndPointsWorld(vtkIdType startCurvePointIndex, vtkIdType endCurvePointIndex);
 
+  void ResampleCurveSurface(double controlPointDistance, vtkMRMLModelNode* node );
+  // Resample a curve with points constrained to surface
```

Comment:
Put comments above the method.

use /// instad of // (that indicates the document generator to include the comment in the class documentation)